src/DataFramesMeta.jl

```diff
@@ -143,6 +155,10 @@ function replace_dotted!(e, membernames)
     Expr(:., x_new, y_new)
 end
 
+getsinglecolumn(df, s::DataFrames.ColumnIndex) = df[!, s]
+getsinglecolumn(df, s) = throw(ArgumentError("Only indexing with Symbols and Strings" *
+    "Is currently allowed with cols"))
```

Comment:
```suggestion
    "is currently allowed with cols"))
```
Also align with above string.